### ISSUE TITLE
PDB structure culstering

### ISSUE THREAD

**Issue by pengzhangzhi** [a-r-j/graphein#314]
Hi @a-r-j and @amorehead, I would like to propose a feature: PDB structure clustering. It would be useful for structure-related tasks like structure prediction and generation. Would you be interested in this idea and want to talk about how to implement this feature? I am thinking about using foldseek for clustering and creating metadata containing the clustering information.

**Comment by pengzhangzhi** [comment]
Hi @amorehead, I try to reach you by email but got no response. It's possible that my email has ended up in their spam folder or that you have not had the chance to respond yet.
Is there any way to reach you in private?

**Comment by pengzhangzhi** [comment]
Yep! Happy to help! I think the first thing is to figure out the exact features we want. I personally have a use case. I want to cluster all pdb structures into N clusters, where N can be very small like 2. In each cluster, we can further cluster them to derive representative samples. Seems like current tools like foldseek does not support that preset num of clusters.

**Comment by a-r-j** [comment]
Hmm, what do you think about an approach where you use FoldSeek to get a set of representative clusters, then you can apply some hierarchical clustering method based on the inter-cluster representative structure TM scores?
